Added Heartbeat-Respiratory-Subsystem.md

Michaelwwest98 opened the pull request:
Originally edited together and uploaded by AndreiM002 somewhere else, Michaelwwest98 has copy/pasted the file and uploaded it to the signoffs file under the project's Github home page.

Michaelwwest98 committed:
Added Heartbeat-Respiratory-Subsystem.md

Originally edited together and uploaded by AndreiM002 somewhere else, Michaelwwest98 has copy/pasted the file and uploaded it to the signoffs file under the project's Github home page.

JesseTNRoberts commented on Documentation/Signoffs/Heartbeat-Respiratory-Subsystem.md:
```diff
@@ -0,0 +1,107 @@
+Specs and Constraints table for signoff:			
+
+
+<table>
```
Don't do this. use markdown syntax where possible.

JesseTNRoberts commented on Documentation/Signoffs/Heartbeat-Respiratory-Subsystem.md:
```diff
@@ -0,0 +1,107 @@
+
+The purpose of this subsystem is to measure the presence of a heartbeat as well as measure the respiratory rate of a person contactlessly. This subsystem will be composed of a singular radar that will perform both of these actions. The measuring of a person’s presence of a heartbeat as well as respiratory rate will be done by measuring the rise and fall rates of a person’s chest to measure the re ...
+
+Specs and Constraints table for signoff:			
```
Good constraints

JesseTNRoberts reviewed (commented):
The design is almost non-existent. This is not a sufficient design.

Michaelwwest98 committed:
Update Heartbeat-Respiratory-Subsystem.md

Updated the buildable wiring schematic and added a few new images for added clarity

JesseTNRoberts commented on Documentation/Signoffs/Heartbeat-Respiratory-Subsystem.md:
```diff
@@ -0,0 +1,112 @@
+Heartbeat/Respiratory Subsystem 
```
Use headers

JesseTNRoberts commented on Documentation/Signoffs/Heartbeat-Respiratory-Subsystem.md:
```diff
@@ -0,0 +1,112 @@
+  <tr>
+   <td>Heartbeat analysis
+   </td>
+   <td>Must sense frequencies from 0.45 Hz to 2.92 Hz
```
Why these numbers?

JesseTNRoberts commented on Documentation/Signoffs/Heartbeat-Respiratory-Subsystem.md:
```diff
@@ -0,0 +1,112 @@
+
+Analysis:
+
+The radar that will be used to accomplish this subsystems function is the Speed MR60BHA1 60GHz mmWave Module. This radar sensor was built for the task of vital detection with the ability to measure heart in the range of 60-100 bpm and respiratory rate of up to 25 times per minute. As specified by the product description the typical accuracy of the breath rate is 90% and the accuracy of the heart  ...
```
Doesn't meet the constraints.

Michaelwwest98 committed:
Update Heartbeat-Respiratory-Subsystem.md

Michaelwwest98 committed:
Update Heartbeat-Respiratory-Subsystem.md

AndreiM002 committed:
Update Heartbeat-Respiratory-Subsystem.md

AndreiM002 committed:
Update Heartbeat-Respiratory-Subsystem.md

JesseTNRoberts commented:
Do not merge. I will merge this when I approve it. Unmerge this PR.
